src/worker/start_geneva.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+ConfigFile="${1}"
+CONTAINER_NAME="genevamdmagent"
+
+# check if the docker container is running
+if docker ps --format '{{.Names}}' | grep -q "^${CONTAINER_NAME}$"; then
```

Comment:
So if the user wants to reconfigure the geneva info (like account name) and rerun the docker, they need to do shut down first? I am fine with this logic as it will not introduce extra errors.